assets/workflowTemplate.yml

```diff
@@ -21,11 +28,18 @@ jobs:
         uses: foundry-rs/foundry-toolchain@v1
         with:
           version: nightly
-          
+
       - name: Run Forge build
-        run: forge build
+        run: |
+          forge --version
+          echo "::group::Contract Sizes"
```

Comment:
Not sure if we need these groups - everything is already contained within a step header that you can collapse

Reply:
🙇‍♂️